automatic/nodejs.install/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,24 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$filePath32 = "$toolsPath\node-v7.4.0-x86.msi"
+$filePath64 = "$toolsPath\node-v7.4.0-x64.msi"
+$installFile = if ((Get-ProcessorBits 64) -and $env:chocolateyForceX86 -ne 'true') {
+                      Write-Host "Installing 64 bit version"; $filePath64 }
+               else { Write-Host "Installing 32 bit version"; $filePath32 }
+
+$packageArgs = @{
+    PackageName    = 'nodejs.install'
+    FileType       = 'msi'
+    SoftwareName   = 'Node.js'
+    File           = $installFile
+    SilentArgs     = "/quiet"
+    ValidExitCodes = @(0)
+}
+Install-ChocolateyInstallPackage @packageArgs
+
+$nodePath = "$env:SystemDrive\Program Files\nodejs"
+if (![System.IO.Directory]::Exists($nodePath)) {$nodePath = "$env:SystemDrive\Program Files (x86)\nodejs";}
+
+$env:Path = "$($env:Path);$nodePath"
```

Comment:
I believe so yes.
But maybe we should get some additional opinions on this.

/cc @chocolatey/chocolateycommunitypackages